Firstassignoops.cpp

```diff
@@ -0,0 +1,341 @@
+#include<iostream>
+#include<list>
+#include<string>
+#include<chrono>
+using namespace std ; 
+
+
+

```

Comment:
I forgot to write this code:
int main (){
    cout<<"What function do you want to implement ?"<<endl;

}